x/marketmaker/client/cli/tx.go

```diff
@@ -0,0 +1,223 @@
+			pairIds := []uint64{}
+			switch len(args) {
+			case 1:
+				pairIdsStr := strings.Split(args[0], ",")
+
+				for _, i := range pairIdsStr {
+					pairId, err := strconv.ParseUint(i, 10, 64)
+					if err != nil {
+						return fmt.Errorf("parse pair id: %w", err)
+					}
+					pairIds = append(pairIds, pairId)
+				}
+			default:
+				return fmt.Errorf("only pool-ids flag must be specified")
+			}
+
+			msg := types.NewMsgApplyMarketMaker(farmer, pairIds)
+
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	flags.AddTxFlagsToCmd(cmd)
+
+	return cmd
+}
+
+// NewClaimIncentives implements the remove plan handler.
+func NewClaimIncentives() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "claim",
+		Args:  cobra.ExactArgs(0),
+		Short: "Claim all claimable incentives",
+		Long: fmt.Sprintf(`
+Claim all market making incentives distributed through governance
+Example:
+$ %s tx %s claim --from mykey`,
+			version.AppName, types.ModuleName,
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			creator := clientCtx.GetFromAddress()
+
+			msg := types.NewMsgClaimIncentives(creator)
+
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+	flags.AddTxFlagsToCmd(cmd)
+	return cmd
+}
+
+// GetCmdSubmitMarketMakerProposal implements the inclusion/exclusion/rejection/distribution for market maker command handler.
+func GetCmdSubmitMarketMakerProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "market-maker-proposal [proposal-file] [flags]",
+		Args:  cobra.ExactArgs(1),
+		Short: "Submit a market maker proposal",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`Submit a market maker proposal along with an initial deposit. You can submit this governance proposal
+to inclusion, exclusion, rejection, and incentive distribution for market maker. The proposal details must be supplied via a JSON file. A JSON file to add request proposal is 
+provided below.
+
+Example:
+$ %s tx gov submit-proposal market-maker-proposal <path/to/proposal.json> --from=<key_or_address> --deposit=<deposit_amount>
+
+Where proposal.json contains:
+
+{
+    "title": "Market Maker Proposal",
+    "description": "Are you ready to market making?",
+    "inclusions":
+    [
+        {
+            "address": "cosmos1vqac3p8fl4kez7ehjz8eltugd2fm67pckpl7pn",
+            "pair_id": "1"
+        }
+    ],
+    "exclusions":
+    [],
+    "rejections":
+    [
+        {
+            "address": "cosmos1vqac3p8fl4kez7ehjz8eltugd2fm67pckpl7pn",
+            "pair_id": "2"
+        }
+    ],
+    "distributions":
+    [
+        {
+            "address": "cosmos1vqac3p8fl4kez7ehjz8eltugd2fm67pckpl7pn",
+            "pair_id": "1",
+            "amount":
+            [
+                {
+                    "denom": "stake",
+                    "amount": "100000000"
+                }
+            ]
+        }
+    ]
+}
```

Comment:
I suggest to format the JSON
```suggestion
{
  "title": "Market Maker Proposal",
  "description": "Are you ready to market making?",
  "inclusions": [
    {
      "address": "cosmos1vqac3p8fl4kez7ehjz8eltugd2fm67pckpl7pn",
      "pair_id": "1"
    }
  ],
  "exclusions": [],
  "rejections": [
    {
      "address": "cosmos1vqac3p8fl4kez7ehjz8eltugd2fm67pckpl7pn",
      "pair_id": "2"
    }
  ],
  "distributions": [
    {
      "address": "cosmos1vqac3p8fl4kez7ehjz8eltugd2fm67pckpl7pn",
      "pair_id": "1",
      "amount": [
        {
          "denom": "stake",
          "amount": "100000000"
        }
      ]
    }
  ]
}
```